components/ai_chat/resources/page/components/main/style.module.scss

```diff
@@ -243,3 +306,7 @@
   // See https://github.com/brave/brave-browser/issues/46042
   margin-bottom: calc(-1 * var(--iframe-additional-margin-for-menus));
 }
+
+.aichatIframeContainer.dragActive {
+  pointer-events: none; // Disable iframe interaction during drag operations
```

Comment:
do we need this? Doesn't displaying an overlay over the top achieve the same thing?